src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/JwtAuthHelper.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.security.oauth2.jwt.JwtDecoder
+import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
+import org.springframework.stereotype.Component
+import java.security.KeyPair
+import java.security.KeyPairGenerator
+import java.security.interfaces.RSAPublicKey
+
+@Component
+class JwtAuthHelper {
```

Comment:
Prob just me not understanding, but I am surprised this is needed. I believe this is the default with JWK set uri?!

Reply:
I think @JamieDraperUK has accidentally put this in `src/main` rather than `src/test` :grin:

Reply:
good spot though @sp-watson